### ISSUE TITLE
4.0.2 build.gradle RN aar finding is broken

### ISSUE THREAD

**Issue by likern** [ospfranco/react-native-quick-sqlite#65]
Hello 👋🏻 
I've tried to upgrade to `4.0.1` from `3.1.7` and got failed build.

```bash
> Configure project :react-native-quick-sqlite
react-native-quick-sqlite: node_modules/ found at: /home/me/Breeffy/feeve-monorepo/packages/app/node_modules
ReactNativeFirebase WARNING: NPM package '@react-native-firebase/auth' depends on '@react-native-firebase/app' v9.0.0 but found v8.4.7, this might cause build issues or runtime crashes.
ReactNativeFirebase WARNING: NPM package '@react-native-firebase/firestore' depends on '@react-native-firebase/app' v9.0.0 but found v8.4.7, this might cause build issues or runtime crashes.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.9/userguide/command_line_interface.html#sec:command_line_warnings
Warning: Mapping new ns http://schemas.android.com/repository/android/common/02 to old ns http://schemas.android.com/repository/android/common/01
Warning: Mapping new ns http://schemas.android.com/repository/android/generic/02 to old ns http://schemas.android.com/repository/android/generic/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/addon2/02 to old ns http://schemas.android.com/sdk/android/repo/addon2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/repository2/02 to old ns http://schemas.android.com/sdk/android/repo/repository2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/sys-img2/02 to old ns http://schemas.android.com/sdk/android/repo/sys-img2/01

FAILURE: Build failed with an exception.

* Where:
Build file '/home/me/Breeffy/feeve-monorepo/packages/app/node_modules/react-native-quick-sqlite/android/build.gradle' line: 190

* What went wrong:
A problem occurred evaluating project ':react-native-quick-sqlite'.
> Expected directory '/home/me/Breeffy/feeve-monorepo/packages/app/node_modules/react-native/android' to contain exactly one file, however, it contains no files.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 8s
```

My `android` folder has the following structure
```bash
[app]$ tree /home/me/Breeffy/feeve-monorepo/packages/app/node_modules/react-native/android
/home/me/Breeffy/feeve-monorepo/packages/app/node_modules/react-native/android
└── com
    └── facebook
        └── react
            └── react-native
                ├── 0.66.1
                │   ├── react-native-0.66.1.aar
                │   ├── react-native-0.66.1.aar.md5
                │   ├── react-native-0.66.1.aar.sha1
                │   ├── react-native-0.66.1.aar.sha256
                │   ├── react-native-0.66.1.aar.sha512
                │   ├── react-native-0.66.1.module
                │   ├── react-native-0.66.1.module.md5
                │   ├── react-native-0.66.1.module.sha1
                │   ├── react-native-0.66.1.module.sha256
                │   ├── react-native-0.66.1.module.sha512
                │   ├── react-native-0.66.1.pom
                │   ├── react-native-0.66.1.pom.md5
                │   ├── react-native-0.66.1.pom.sha1
                │   ├── react-native-0.66.1.pom.sha256
                │   └── react-native-0.66.1.pom.sha512
                ├── maven-metadata.xml
                ├── maven-metadata.xml.md5
                ├── maven-metadata.xml.sha1
                ├── maven-metadata.xml.sha256
                └── maven-metadata.xml.sha512

5 directories, 20 files
```

**Comment by likern** [comment]
I think it's not related to monorepo structure itself, but anyway mentioned this too.

**Comment by ospfranco** [comment]
Ah, probably has something to do with the [patch for RN 0.69](https://github.com/ospfranco/react-native-quick-sqlite/commit/b4f7bec1e4db3a18f35174b97d93c1a8361bac52) that was applied last week... if you can, you update to RN 0.69+ if not, maybe I can take a look (much) later

**Comment by ospfranco** [comment]
Fixed under 4.0.3

Please do consider sponsoring! I have to take time of paid projects to fix this kind of issues
